src/components/global/Playground/index.tsx

```diff
@@ -129,11 +165,42 @@ export default function Playground({
     const codeSnippets = {};
     Object.keys(code).forEach((key) => {
       // Instantiates the React component from the MDX content.
-      codeSnippets[key] = code[key]({});
+      if (typeof code[key] === 'function') {
+        codeSnippets[key] = code[key]({});
+      } else if (typeof code[key] === 'object') {
+        const fileSnippets = {};
+        for (const fileName of Object.keys(code[key].files)) {
+          fileSnippets[`${fileName}`] = code[key].files[fileName]({});
+        }
```

Comment:
What does this do? I know it "Instantiates the React component from the MDX content" but not sure why the extra cases are needed.

Reply:
I can add a comment block in here explaining this code. 

If the value of `code[key]` is a function, that means you are using a single-file example and are binding the MDX content directly to the prop:

```tsx
import javascript from 'javascript.md';

<Playground code={{ javascript }} />
```

Otherwise, if the type is an object, you are using the advanced configuration object to specify multiple code snippets for a single usage target and have options for customizing the angular `AppModule`. It loops through all your specified files and does the same thing as the single-file example, which constructs the React component from the MDX object. We keep track of the file names, because they coincide with the path of the file when creating the Stackblitz example from the code snippets.

```tsx
import file_one from 'file-one.md';
import file_two from 'file-two.md';

<Playground code={{
  javascript: {
    files: {
      'src/example/file-one.html': file_one,
      'src/example/file-two.html': file_two
    }
  }
}} />
```